intelmq/tests/lib/utils.py

```diff
@@ -0,0 +1,64 @@
+from intelmq.lib import utils
+import unittest
+import tempfile
+import re
+import os.path
+import StringIO
```

Comment:
Using `import io.StringIO` or `from io import StringIO` is Python3-proof. This is not a requirement currently, but will make it easier later.